plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/Prism/Mirror.hs

```diff
@@ -31,17 +31,20 @@ import qualified Ledger.Constraints                                          as
 import           Ledger.Crypto                                               (PubKeyHash, pubKeyHash)
 import qualified Ledger.Typed.Scripts                                        as Scripts
 import           Ledger.Value                                                (TokenName)
+import           Schema                                                      (ToSchema)
+import           Wallet.Emulator                                             (walletPubKey)
+import           Wallet.Emulator.Wallet                                      (Wallet)
 
 -- | Reference to a credential tied to a specific owner (public key address).
 --   From this, and the public key of the Mirror instance, we can compute the
 --   address of the state machine script that locks the token for the owner.
 data CredentialOwnerReference =
     CredentialOwnerReference
         { coTokenName :: TokenName
-        , coOwner     :: PubKeyHash
+        , coOwner     :: Wallet
```

Comment:
Yeah that's the reason, I wanted to avoid having to copy & paste the wallets' public keys everywhere. But this change isn't very effective because you still need the keys in other places.

> the application should deal with public keys, and the UI should handle mapping those to "wallets" or known identities.

Yes I agree